ci/cases/pr/C96C48_hybatmsoilDA.yaml

```diff
@@ -0,0 +1,24 @@
+experiment:
+  system: gfs
+  mode: cycled
+
+arguments:
+  pslot: {{ 'pslot' | getenv }}
+  app: ATM
+  resdetatmos: 96
+  resensatmos: 48
+  comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
+  expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
+  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C96C48/20241120
+  idate: 2021122012
+  edate: 2021122100
+  nens: 2
+  interval: 24
+  start: cold
+  yaml: {{ HOMEgfs }}/ci/cases/yamls/atmsoilDA_defaults_ci.yaml
+
+skip_ci_on_hosts:
```

Comment:
@ClaraDraper-NOAA are these tests skipped to save time or because things do not work on these other machines? Particularly concerned about orion, hercules and gaeac6 which are being considered for GFSv17 retrospectives.

Reply:
To save time. I've turned the tests on for all machines.

Reply:
Thanks for the explanation @ClaraDraper-NOAA   and confirming that it should work everywhere.